packages/components/molecules/f-tabs/stories/Tabs.stories.js

```diff
@@ -37,7 +37,7 @@ export const VueTabsComponent = (args, { argTypes }) => ({
 VueTabsComponent.storyName = 'f-tabs';
 VueTabsComponent.args = {
     selected: 'a',
-    tabTitle: 'I can be renamed'
+    tabTitle: 'How it works'
```

Comment:
As the assertion is an `.includes()` this should work with:
```
    tabTitle: 'How it works (I can be renamed)';
```
If you want to keep the storybook sugar.

Reply:
Good suggestion, I like it.